src/Stratis.Bitcoin.Features.Wallet.Tests/WalletControllerTest.cs

```diff
@@ -1352,7 +1352,7 @@ public void SendTransactionSuccessfulReturnsWalletSendTransactionModelResponse()
 
             var mockWalletWrapper = new Mock<IBroadcastManager>();
             mockWalletWrapper.Setup(m => m.TryBroadcastAsync(transaction))
-                .Returns(Task.FromResult(Success.Yes));
+                .Returns(Task.FromResult(true));
```

Comment:
If Success.Yes == true and Success.No == false, how about Success.DontKnow?

Reply:
There is no Success.DontKnow anymore. 
In previews version if we were getting Success.DontKnow from the LightWalletBroadcastManager we would start waiting for 21 sec for tx to be propagated. If it's not we assume it's false and throw. 
In my implementation I moved that 21 sec waiting to the LightWalletBroadcastManager itself because this behavior only makes sense for the light node. And that's why I was able to get rid of the Success.DontKnow.